keylime/da/examples/rekor.py

```diff
@@ -0,0 +1,51 @@
+import shutil
+
+from keylime import keylime_logging
+from keylime.cmd_exec import run
+
+# setup logging
+logger = keylime_logging.init_logging("transparency_log")
+
+
+def record_signature_create(record_object, agent_data, ts_url, cert_tls_pub, tmp_dir=None):
+    logger.debug("Uploading signature entry for data referring to agent %s on rekor", agent_data["agent_id"])
+    command = [
+        "rekor-cli",
+        "upload",
+        "--artifact",
+        record_object["contents_file_path"],
+        "--rekor_server",
+        ts_url,
+        "--signature",
+        record_object["signature_file_path"],
+        "--pki-format",
+        "x509",
+        "--public-key",
+        cert_tls_pub,
+    ]
+    run(cmd=command)
+
+    if tmp_dir:
```

Comment:
IMHO `tmp_dir` should not be a parameter. Should be managed (created, used and removed) inside this class.

Reply:
For safety maybe better check if the rekor-cli is present?

Reply:
Perhaps `tempfile.TemporaryDirectory` might be safer.